mysql-test/suite/tokudb/t/dir_per_db_rename_with_non_alnum_chars.test

```diff
@@ -0,0 +1,22 @@
+--source include/have_tokudb.inc
```

Comment:
Hmm, sounds reasonable enough, particularly since the size and scope of this test is pretty small.